### ISSUE TITLE
[CSS] containment docs tasks, Fx 70 (high priority)

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/sprints#2106]
Trello: https://trello.com/c/AQOpkXpT/3-css-containment-fx-70

# Acceptance criteria

1. - [x] [Enable pref for CSS Containment (layout.css.contain.enabled)](https://bugzilla.mozilla.org/show_bug.cgi?id=1487493)
2. - [x] Usage tutorial / examples are needed

**Comment by rachelandrew** [comment]
There was no CSS Containment page so I have created one with explanations of usage for CSS Containment. I have also added an example to the property page.

- https://developer.mozilla.org/en-US/docs/Web/CSS/contain
- https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment

The bug says Firefox 70, however it looks like it shipped in 69 and the compat data has been updated to say as such. So unless that's incorrect I don't think there is anything to do there.
